apisix/cli/ops.lua

```diff
@@ -105,6 +105,7 @@ local function local_dns_resolver(file_path)
     for line in file:lines() do
         local addr, n = line:gsub("^nameserver%s+(%d+%.%d+%.%d+%.%d+)%s*$", "%1")
         if n == 1 then
+            io.stderr:write("DNS: ",addr,"\n")
```

Comment:
why do we use `io.stderr` here? is it an error case?

Reply:
already removed 😁